Pictures/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+
+function(preparing_benchmark arg0)
+    add_custom_target(make-directory-benchmark ALL
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_BINARY_DIR}/bin/Benchmark)
+
+    set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin/Benchmark)
+
+    file(COPY ${IMAGES_DIR}/BITMAPA.png
+        DESTINATION ${CMAKE_BINARY_DIR}/bin/Benchmark/Pictures/)
+endfunction()
+
+function(preparing_singlePoint_tests arg0)
```

Comment:
This and other functions do not look as a utility function. It seems to be rather something target specific.
Please consider to make it more generic or move it to target where belongs to.